docs/current_docs/cloud/572923-get-started.mdx

```diff
@@ -36,58 +36,65 @@ At the end of this step, you will have signed up for Dagger Cloud and obtained a
-:::
+Once you have your token, you can now use it to send telemetry from your CI and development machines.
 
-To connect your Dagger Cloud account with a CI provider or CI tool, you must first obtain a Dagger Cloud token.
+### Local development
+You can visualize and debug your local Dagger pipeline runs with Dagger Cloud to identify issues before pushing them to CI.
 
-Browse to the *Organizations Settings* page of the Dagger Cloud dashboard (accessible by clicking your user profile icon in the Dagger Cloud interface). Select your organization and navigate to the *Configuration* tab. Note the Dagger Cloud token carefully.
+To configure local development, set your Dagger Cloud token as an environment variable. For example: `export DAGGER_CLOUD_TOKEN={your token}`
 
-    ![Get token](/img/current_docs/cloud/get-started/get-token.png)
+:::info BEST PRACTICES FOR VISUALIZING LOCAL RUNS IN DAGGER CLOUD
+Dagger Cloud uses metadata from your version control system (VCS), such as git, to populate some information in the user interface. Therefore, you should run your Dagger pipelines within a VCS context, such as a local directory linked to a cloned git repository.
+:::
 
-Once you have a token, the general procedure to connect Dagger Cloud with a CI provider/CI tool is:
 
-- Store the token as a secret with your CI provider/in your CI tool.
-- Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
-- For *Team* plan subscribers with ephemeral CI runners only: Adjust the Docker configuration to wait longer  before killing the Dagger Engine container, to give it more time to push data to the Dagger Cloud cache
+### In CI
+The general procedure to connect Dagger Cloud with a CI provider/CI tool is:
 
-:::danger
-You must store the Dagger Cloud token as a secret (not plaintext) with your CI provider and reference it in your CI’s workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project.
-:::
+- Add your Dagger Cloud token to your CI workflows.
+  - Store the Dagger Cloud token as a secret with your CI.
+    :::danger Keep your Dagger Cloud token private
+    You must store the Dagger Cloud token as a secret (not plaintext) with your CI provider and reference it in your CI’s workflow. Using a secret is recommended to protect your Dagger Cloud account from being used by forks of your project. We provide links in the steps below for configuring secrets with popular CI tools.
+    :::
+  - Add the secret to your CI environment as a variable named `DAGGER_CLOUD_TOKEN`.
+- For *Team* plan subscribers with ephemeral CI runners only: Add a wait step for the Docker configuration to allow the Dagger Engine to push cache volumes to the experimental Dagger Cloud distributed cache. Subscribers on the *Individual* plan do not have access to the experimental distributed cache and do not need to add this step.
```

Comment:
```suggestion
- For *Team* plan subscribers with ephemeral CI runners only: Add a wait step (example below) for the Docker configuration to allow the Dagger Engine to push cache volumes to the experimental Dagger Cloud distributed cache. Subscribers on the *Individual* plan do not have access to the experimental distributed cache and do not need to add this step.
```

Reply:
Updated in the 4 instances where this is mentioned